src/Microsoft.WSMan.Management/CredSSP.cs

```diff
@@ -725,23 +706,15 @@ private void UpdateGPORegistrySettings(string applicationname, string[] delegate
                 //open the registry key.If key is not present,create a new one
                 Credential_Delegation_Key = rootKey.OpenSubKey(Registry_Path_Credentials_Delegation, true);
                 if (Credential_Delegation_Key == null)
-                    Credential_Delegation_Key = rootKey.CreateSubKey(Registry_Path_Credentials_Delegation
-#if !CORECLR
-                        , RegistryKeyPermissionCheck.ReadWriteSubTree
```

Comment:
`Microsoft.Win32.RegistryKeyPermissionCheck` is available in .NET Core. Maybe we should keep it?